homeworks/roman.vlasov_batteryfree/homework_markup_1/style.css

```diff
@@ -0,0 +1,110 @@
+body {
+  background-color: #FFF;
+  font-family: "Open Sans", "sans-serif";
+  width: 1024px;
+  margin: auto;
+}
+
+/* .title-page -> height: 50ch; -> lint say Unexpected unit "ch" (unit-whitelist)*/
+
+.title-page {
+  background-image: url("images/titlelayers.jpg");
+  background-repeat: no-repeat;
+  background-position: top;
+  background-size: 1024px auto;
+  height: 500px;
+}
+
+.logo {
+  padding-top: 40px;
+  padding-left: 40px;
+}
+
+.navigation {
+  margin-top: -40px;
+  margin-left: 45%;
+}
+
+.link {
+  display: flex;
+  list-style-type: none;
+}
+
+.link-style {
+  padding-left: 40px;
+  color: #FFF;
+  font-size: 10px;
+  font-weight: 290;
```

Comment:
double check about font-weight property
in most cases it's used with such values:

100 - light
400 - regular
600 - bold
900 - heavy

290 is not a by standard value here